SimPEG/electromagnetics/static/induced_polarization/simulation.py

```diff
@@ -40,6 +39,25 @@ def sigmaDeriv(self):
     def rhoDeriv(self):
         return sp.diags(self.rho) @ self.etaDeriv
 
+    @property
```

Comment:
do not make this a visible property to the user, it should be hidden, likely just need to make sure fields are called before a getJ